docker/base-deps/README.md

```diff
@@ -13,7 +13,6 @@ Images are `tagged` with the format `{Ray version}[-{Python version}][-{Platform
 | `latest`                     | The most recent Ray release. |
 | `x.y.z`                      | A specific Ray release, e.g. 1.12.1 |
 | `nightly`                    | The most recent Ray development build (a recent commit from Github `master`) |
-| `6 character Git SHA prefix` | A specific development build (uses a SHA from the Github `master`, e.g. `8960af`). |
```

Comment:
nit: since we are here, change `Github` into `GitHub` (capital `H`) in these pages (or maybe in another PR).

Reply:
done! also updated it to capital H for other readme files